File: SoalA/hardware.java
public class hardware extends product {
    /* class hardware adalah child dari class product */

    /* atribut privat */
    private String model;
    private String brand;

    /* membuat constructor */
    public hardware(String model, String brand) {
        this.model = model;
        this.brand = brand;
    }

    public hardware() {

    }

    /* Get-Set Methods model */
    public void setModel(String model) {
        this.model = model;
    }

    public String getModel() {
        return model;
    }

    /* Get-Set Methods brand */
    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getBrand() {
        return brand;
    }
}

File: SoalB/Pyhton/hardware.py
from product import product


class hardware(product):
    # class hardware adalah child dari class product

    # atribut private
    __model = ""
    __brand = ""

    # membuat constructor
    def __init__(self):
        self.__model = ""
        self.__brand = ""

    # get-set methods model
    def setModel(self, model):
        self.__model = model

    def getModel(self):
        return self.__model

    # get-set methods brand
    def setBrand(self, brand):
        self.__brand = brand

    def getBrand(self):
        return self.__brand
